### ISSUE TITLE
Dummy Debian packages

### ISSUE THREAD

**Issue by Mebus** [open62541/open62541#3914]
Hallo,

the Debian packages under "Releases" seem to be dummies:

* libopen62541_dev.deb
* libopen62541_tools.deb

The first one only contains the string "Test file1" and the second one "Test file2". 

They should either contain a valid Debian package or they should not exist.

Thanks

Greetings

Mebus

**Comment by trensetim** [comment]
Agreed. Just stupidly tried to install them on a sandbox dev env and got confused

**Comment by teccheck** [comment]
Official Debian packages are created and are available soon
